view/script/patient.js

```diff
@@ -38,7 +39,9 @@
         }
     };
 
+    var allDatesConfig = jQuery.extend(true, {}, window.dates);
```

Comment:
nevermind this is a recursive merge of dates to a plain object? still :confused: